Validation/CTPPS/python/simu_config/year_2016_cff.py

```diff
@@ -12,11 +12,12 @@
 # geometry
 from Geometry.VeryForwardGeometry.geometryRPFromDD_2017_cfi import * # using 2017 here is OK
 ctppsCompositeESSource.compactViewTag = ctppsGeometryESModule.compactViewTag
+ctppsCompositeESSource.isRun2 = True
 del ctppsGeometryESModule # this functionality is replaced by the composite ES source
```

Comment:
The preferred way to achieve this would be
```py
from Geometry.VeryForwardGeometry.geometryRPFromDD_2017_cfi import ctppsGeometryESModule as _ctppsGeometryESModule
... = _ctppsGeometryESModule.compactViewTag
```
Variables starting with underscore do not propagate with `import *` or `process.load()`.